### PR TITLE
Add CVEs for two Metasploit modules

### PR DESCRIPTION
**Pull request by todb-r7**
Covers the issues discussed at rapid7/metasploit-framework#14008 and rapid7/metasploit-framework#14015

@bcoles feel free to take a look and tell me if something is grossly wrong! I always feel like I'm just guessing at CVSS scores, they get rewritten upstream anyway by NVD (nobody believes the issuer of CVEs for CVSS, turns out).

### CODE REVIEW

File: 2020/7xxx/CVE-2020-7377.json [hunk 1]
```diff
@@ -1,18 +1,105 @@
...
-    "data_format": "MITRE",
-    "data_version": "4.0",
     "CVE_data_meta": {
+        "ASSIGNER": "cve@rapid7.com",
+        "DATE_PUBLIC": "2020-08-18T08:48:00.000Z",
         "ID": "CVE-2020-7377",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
+        "STATE": "PUBLIC",
+        "TITLE": "Rapid7 Metasploit Framework Pelative Path Traversal in telpho10_credential_dump module"
```
Comment (djordan-r7) [inline 2020/7xxx/CVE-2020-7377.json]: "Pelative Path Traversal" - should be Relative Path Traversal?

**Review by djordan-r7 — CHANGES_REQUESTED** [review]
Just one copypaste error!

File: 2020/7xxx/CVE-2020-7377.json (hunk 1 above)
Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7377.json]: Fixed!

File: 2020/7xxx/CVE-2020-7377.json [hunk 2]
```diff
@@ -1,18 +1,105 @@
...
+                                    "version_data": [
+                                        {
+                                            "version_affected": ">=",
+                                            "version_name": "4.12.40",
+                                            "version_value": "4.12.40"
+                                        },
+                                        {
+                                            "version_affected": "<",
+                                            "version_name": "6.0.2",
+                                            "version_value": "6.0.2"
```
Comment (bcoles) [inline 2020/7xxx/CVE-2020-7377.json]: This issue has not yet been fixed. Presuming that my open PR (https://github.com/rapid7/metasploit-framework/pull/14034) is reviewed, found to be sufficient, and merged this week, the versions affected would be `< 6.0.3`.

File: 2020/7xxx/CVE-2020-7377.json [hunk 3]
```diff
@@ -1,18 +1,105 @@
...
+                "name": "https://github.com/rapid7/metasploit-framework/issues/14015",
+                "refsource": "CONFIRM",
+                "url": "https://github.com/rapid7/metasploit-framework/issues/14015"
+            }
+        ]
+    },
+    "solution": [
+        {
+            "lang": "eng",
+            "value": "Users should update to version 6.0.2 or later of the Metasploit Framework."
```
Comment (bcoles) [inline 2020/7xxx/CVE-2020-7377.json]: `6.0.3` as above.

File: 2020/7xxx/CVE-2020-7377.json (hunk 2 above)
Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7377.json]: Ah, thanks. Will update both!

**Review by todb-r7 — COMMENTED** [review]
Updated!

**Comment by todb-r7** [comment]
Man all these fancy Github review buttons. Never mind! All edits made, gtg now. Thanks!

File: 2020/7xxx/CVE-2020-7377.json (hunk 2 above)
Comment (bcoles) [reply 2020/7xxx/CVE-2020-7377.json]: No need to update both.

CVE-2020-7376 was resolved in 6.0.2.

CVE-2020-7377 has now been patched and the patch will be available in the 6.0.3 release this week.

**Pull request merged**